src/Foundation/NSMetadataItem.cs

```diff
@@ -177,5 +187,1045 @@ bool GetBool (NSString key)
+		public NSString Producer {
+			get {
+				return Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.ProducerKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public NSString Genre {
+			get {
+				return Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.GenreKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public NSString [] Performers {
+			get {
+				using (var a = Runtime.GetNSObject<NSArray> (GetHandle (NSMetadataQuery.PerformersKey)))
+					return NSArray.FromArray<NSString> (a);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public NSString OriginalFormat {
+			get {
+				return Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.OriginalFormatKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public NSString OriginalSource {
+			get {
+				return Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.OriginalSourceKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public NSString [] AuthorEmailAddresses {
+			get {
+				using (var a = Runtime.GetNSObject<NSArray> (GetHandle (NSMetadataQuery.AuthorEmailAddressesKey)))
+					return NSArray.FromArray<NSString> (a);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public NSString [] RecipientEmailAddresses {
+			get {
+				using (var a = Runtime.GetNSObject<NSArray> (GetHandle (NSMetadataQuery.RecipientEmailAddressesKey)))
+					return NSArray.FromArray<NSString> (a);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public NSString [] AuthorAddresses {
+			get {
+				using (var a = Runtime.GetNSObject<NSArray> (GetHandle (NSMetadataQuery.AuthorAddressesKey)))
+					return NSArray.FromArray<NSString> (a);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public NSString [] RecipientAddresses {
+			get {
+				using (var a = Runtime.GetNSObject<NSArray> (GetHandle (NSMetadataQuery.RecipientAddressesKey)))
+					return NSArray.FromArray<NSString> (a);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public bool IsLikelyJunk {
+			get {
+				return GetBool (NSMetadataQuery.IsLikelyJunkKey);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public NSString [] ExecutableArchitectures {
+			get {
+				using (var a = Runtime.GetNSObject<NSArray> (GetHandle (NSMetadataQuery.ExecutableArchitecturesKey)))
+					return NSArray.FromArray<NSString> (a);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public NSString ExecutablePlatform {
+			get {
+				return Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.ExecutablePlatformKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public NSString [] ApplicationCategories {
+			get {
+				using (var a = Runtime.GetNSObject<NSArray> (GetHandle (NSMetadataQuery.ApplicationCategoriesKey)))
+					return NSArray.FromArray<NSString> (a);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public bool IsApplicationManaged {
+			get {
+				return GetBool (NSMetadataQuery.IsApplicationManagedKey);
+			}
+		}
+#endif
```

Comment:
So, there are a few properties that return NSString or NSString[], why don't we return string and string[]?